### DISCUSSION TITLE
Background colours on forms.

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#13] [General]
Hi all,

It would be a good idea to make the background colour of the forms consistent.

The background colour of the Main Menu is: "#FFD5D8DC", so if we could use that, that would be good.

The colour of the buttons is: "#33000000" so if you want to make the colour of the buttons on your forms
the same, that would be good.

However, if you want to use different colors, that is up to you. We're not going to be assessed on the 
colours we use on our forms.

Also, please keep screenshots of everything you do (even if it seems superfluous to requirements) and
document all testing you carry out.

Thanks everyone,

Mark

**Comment by StephenHillenbrand** [comment]
I agree here and have changed the colours for the background and button
